### ISSUE TITLE
Custom Profile Data Demo Doesn't Work

### ISSUE THREAD

**Issue by rdegges** [stormpath/stormpath-express-react-example#17]
If I try to click the 'Custom Profile Data' link on the example dashboard, it takes me to edit my first name, last name, email, and password. This doesn't showcase any custom data stuff, which it should.

http://g.recordit.co/MAG6Ueirgt.gif

**Comment by rorheden** [comment]
Yeah, I fully agree. The reason why it is missing is because the SDK does not have full support for custom data yet.

There's an issue logged for it:
https://github.com/stormpath/stormpath-sdk-react/issues/36

**Comment by rorheden** [comment]
This is in progress. See: https://github.com/stormpath/stormpath-sdk-react/pull/111

**Comment by robertjd** [comment]
@typerandom I already merged stormpath/stormpath-sdk-react#111, but now I'm seeing some problems when I'm trying to use stormpath-sdk-react/master with the `reproduce-custom-data` branch on this library (which I thought we could merge, to solve this need for custom data examples).

The problem I'm seeing is that the custom data field is being POST-ed like this:

```
{
  "customData.color":"test","password":""
}
```

Whereas it should be like this:

```
{
  "customData": {
    "color": "test"
  }
}
```

The result is that server.js errors because it can't find the custom data object in the body of the request.  I'm currently trying to find the place in the React SDK that is responsible for constructing this post, I'll update this post if I find it before my day is done.

**Comment by robertjd** [comment]
Blarg.. nevermind me :)  I had linked to sdk-react/master incorrectly.  This is actually working fine.  I'm going to issue a PR to bring `reproduce-custom-data` into master.
